src/setup/wizardNames.ts

```diff
@@ -0,0 +1,144 @@
+export const guildNames: string[] = [
+    "Arcane Sanctum of Astoria",
+    "Order of the Crystal Arcanum",
+    "Celestial Academy of Sorcery",
+    "Guild of the Arcane Ember",
+    "Mystic Consortium of Eldoria",
+    "Astral Academy of Enchantment",
+    "Covenant of the Silver Moon",
+    "Sanctum of the Emerald Flame",
+    "Academy of Mystical Lore",
+    "Guild of the Crimson Spell",
+    "Council of Arcane Wisdom",
+    "Institute of Arcane Arts",
+    "Guildhall of the Sapphire Star",
+    "Circle of the Eternal Flame",
+    "Society of the Golden Glyph",
+    "Sanctum of the Verdant Grove",
+    "Consortium of Astral Magi",
+    "Order of the Shadowed Veil",
+    "Institute of Celestial Magics",
+    "Guild of the Ancient Runes",
+    "Arcanum Sanctum of Everfall",
+    "Council of the Astral Serpent",
+    "Academy of the Azure Skies",
+    "Covenant of the Sacred Grove",
+    "Sanctum of the Obsidian Moon",
+    "Circle of the Ethereal Veil",
+    "Guild of the Whispering Winds",
+    "Society of the Twilight Arcanum",
+    "Celestial Sanctum of Astraea",
+    "Institute of Arcane Illumination",
+    "Order of the Crimson Phoenix",
+    "Guild of the Silver Shroud",
+    "Covenant of the Twilight Star",
+    "Academy of the Ethereal Essence",
+    "Sanctum of the Ruby Dawn",
+    "Consortium of the Arcane Citadel",
+    "Circle of the Mystic Grove",
+    "Guild of the Whispering Shadow",
+    "Society of the Ivory Tower",
+    "Institute of the Eternal Flame",
+    "Order of the Astral Phoenix",
+    "Sanctum of the Moonlit Whisper",
+    "Academy of the Verdant Arcana",
+    "Guild of the Celestial Phoenix",
+    "Covenant of the Sapphire Serpent",
+    "Society of the Astral Whisper",
+    "Circle of the Emerald Eye",
+    "Institute of the Crimson Tide",
+    "Order of the Ethereal Ember",
+    "Sanctum of the Celestial Sentinel"
+];
+
+export const fireWizardNames: string[] = [
```

Comment:
Might be good to make sure every name has different initials or starting chars, so in the wizards tabs we can just have (PF), (IF), (EB), etc.

Reply:
Yeah good idea, I'll keep a note of it for the post jam work.